src/common/options.cc

```diff
@@ -4988,6 +4988,12 @@ std::vector<Option> get_global_options() {
     Option("target_max_misplaced_ratio", Option::TYPE_FLOAT, Option::LEVEL_BASIC)
     .set_default(.05)
     .set_description("Max ratio of misplaced objects to target when throttling data rebalancing activity"),
+
+    Option("osdc_thread_count", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
+    .set_default(8)
```

Comment:
i want to believe that 1 or 2 threads would perform just as well as 8 here, but i think we'd need to confirm that with benchmarks

Reply:
I've set the default to 2, for now.